Documentation/git-update-microsoft-git.txt

```diff
@@ -0,0 +1,24 @@
+git-update-microsoft-git(1)
+===========================
+
+NAME
+----
+git-update-microsoft-git - Update the installed version of Git
+
+
+SYNOPSIS
+--------
+[verse]
+'git update-microsoft-git'
+
+DESCRIPTION
+-----------
+This version of Git is based on the Microsoft fork of Git, which
+has custom capabilities focused on supporting monorepos. This
```

Comment:
Should we add a short statement about GVFS here?  Our enhancements in this fork fall into 2 groups: monorepo stuff we want to eventually upstream, and the GVFS-related stuff which will never make it upstream.

Basically, I'm thinking about a short ack/reminder here about why we have this fork at all.

Reply:
We should update the `README` with these details. A user won't see this documentation unless they are looking at the version that is installed with the `microsoft/git` fork.